inst/metadata/time_series_plot_metadata.csv

```diff
@@ -13,6 +13,7 @@ anc_clients,t_(ANC_CLIENTS),t_(ANC_CLIENTS_DESC),,"""0,0"""
 anc_tested,t_(ANC_TESTED),t_(ANC_TESTED_DESC),,"""0,0"""
 anc_tested_pos,t_(ANC_TESTED_POS),t_(ANC_TESTED_POS_DESC),,"""0,0"""
 anc_known_pos,t_(ANC_KNOWN_POS),t_(ANC_KNOWN_POS_DESC),,"""0,0"""
+births_clients_ratio,t_(BIRTHS_CLIENTS_RATIO),t_(BIRTHS_CLIENTS_RATIO_DESC"),,"""0%"""
```

Comment:
This should be a count not percentage?

Reply:
We've chosen to show other ratios as %, e.g. art_adult_sex_ratio and art_adult_child_ratio - I'm happy to change this to a count but just wanted to double check before doing so